doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/configuring-security-aspects.md

```diff
@@ -1 +1,522 @@
-# To be replaced with content
+# Configuring security aspects
+
+## Starting with Keyrock
+
```

Comment:
A brief description of what is keyrock is needed. A link to the component needed